### PR TITLE
[FIX] FIxes bug #79

### PR DESCRIPTION
**Pull request by hurrinico**
In order to solve https://github.com/OCA/stock-logistics-workflow/issues/79 i create this PR

### CODE REVIEW

**Comment by eLBati** [comment]
:+1:

**Comment by lmignon** [comment]
@hurrinico It should be nice to have a unittest that demonstrate your fix..

File: stock_picking_invoice_link/stock.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def _create_invoice_line_from_vals(
         inv_line_id = super(stock_move, self)._create_invoice_line_from_vals(
             cr, uid, move, invoice_line_vals, context=context)
         move.write({'invoice_line_id': inv_line_id})
+        move.picking_id.write({'invoice_id': invoice_line_vals['invoice_id']})
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/stock.py]: You can do a shorter form:

```
move.picking_id.invoice_id = invoice_line_vals['invoice_id']
```

**Comment by hurrinico** [comment]
@lmignon i'm quite busy at the moment, so i can't do the unittest by now.

**Comment by lepistone** [comment]
Hi @hurrinico. Thanks for the fix!

 Looking at the code I see a line added and a method deleted, but I do not understand how this fixes the bug. A agree we need a test here.

Also, the commit messages say "fix" 12 times, but instead it would be great to have only one commit with a message that actually says why you are doing this. Thanks!

**Comment by chafique-delli** [comment]
I tested the fix, it fixes the bug that I also found. :+1:

**Comment by guewen** [comment]
I guess a method has been renamed. Would be cool you if could you squash your commits and put an explanation in the commit message.

**Comment by hurrinico** [comment]
@lepistone @guewen @chafique-delli i've sqhashed all the commit in one and rebased the brach

**Pull request merged**